src/ui/image/Image.tsx

```diff
@@ -38,18 +38,14 @@ const Image = ({ lowQuality, fallback, src, className, ...props }: Image.Props)
 
     return (
         <div className={`${csx.img} ${className}`}
-            style={{ height: props.height, width: props.width }}>
-            {((url === lowQuality) && (error === false))
-                ? <Loader className={`${csx.mask}`} />
-                : ""}
-            {error === true
-                ? <Button className={`${csx.button} ${csx.mask}`} onClick={() => loadImage()}>Retry</Button>
-                : ""}
+            style={{ height: height, width: width }}>
+            {((url === lowQuality) && !error) && <Loader className={`${csx.mask}`} />}
+            {error && <Button className={`${csx.button} ${csx.mask}`} onClick={() => loadImage()}>Retry</Button>}
```

Comment:
You can simply pass function def as onClick prop so no need to do it with anonymous fn `onClick={loadImage}` - but keep in mind that typedefs should always be compliant

Reply:
done